mysql-test/suite/binlog/t/percona_bug_ps290-master.opt

```diff
@@ -0,0 +1,2 @@
+--log-bin
```

Comment:
Is the option file required?

Reply:
No, looks like it's not needed, I removed the file.